test/controllers/CreditAndRefundControllerSpec.scala

```diff
@@ -197,6 +198,82 @@ class CreditAndRefundControllerSpec extends MockAuthenticationPredicate with Moc
 
         status(result) shouldBe Status.SEE_OTHER
       }
+    }*/
+
+    "Controller should start refund process" when {
+      "RepaymentJourneyModel is returned" in new Setup {
+        disableAllSwitches()
+        enable(CreditsRefundsRepay)
+
+        mockSingleBISWithCurrentYearAsMigrationYear()
+        setupMockAgentAuthRetrievalSuccess(testAgentAuthRetrievalSuccess)
+        setupMockAuthRetrievalSuccess(BaseTestConstants.testAuthSuccessWithSaUtrResponse())
+
+        when(mockCreditService.getCreditCharges()(any(), any()))
+          .thenReturn(Future.successful(List(financialDetailCreditAndRefundCharge)))
+        when(mockRepaymentService.start(any(), any())(any()))
+          .thenReturn(Future.successful(RepaymentJourneyModel("/test/url")))
+
+        val result: Future[Result] = controller.startRefund()(fakeRequestWithActiveSession)
+
+        status(result) shouldBe (303)
+      }
+    }
+
+    "Controller should not start refund process" when {
+      "RepaymentJourneyErrorResponse is returned" in  new Setup {
+        disableAllSwitches()
+        enable(CreditsRefundsRepay)
+
+        mockSingleBISWithCurrentYearAsMigrationYear()
+        setupMockAgentAuthRetrievalSuccess(testAgentAuthRetrievalSuccess)
+        setupMockAuthRetrievalSuccess(BaseTestConstants.testAuthSuccessWithSaUtrResponse())
+
+        when(mockCreditService.getCreditCharges()(any(), any()))
+          .thenReturn(Future.successful(List(financialDetailCreditAndRefundCharge)))
+        when(mockRepaymentService.start(any(), any())(any()))
+          .thenReturn(Future.successful(RepaymentJourneyErrorResponse(401, "Token expired")))
+
+        val result: Future[Result] = controller.startRefund()(fakeRequestWithActiveSession)
+
+        status(result) shouldBe 500
```

Comment:
please rely on the values provides in the StandardValues.scala ~ Status trait instead of using "magic numbers"